### PR TITLE
KIN-2 implemented UITabBarController and UINavigationController

### PR DESCRIPTION
**Pull request by niewyspana**
https://misspolazava.atlassian.net/browse/KIN-2

### CODE REVIEW

File: Kino/AppDelegate.swift [hunk 1]
```diff
@@ -7,30 +7,42 @@
...
+        ticket.tabBarItem = UITabBarItem(title: nil, image: UIImage(systemName: "ticket"), tag: 1)
 
-    func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
-        // Called when the user discards a scene session.
-        // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
-        // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
+        let fav = FavouritesViewController(nibName: "FavouritesViewController", bundle: nil)
+        fav.tabBarItem = UITabBarItem(title: nil, image: UIImage(systemName: "heart.fill"), tag: 2)
+
+
```
Comment (Darya-Astapava) [inline Kino/AppDelegate.swift:16]: extra line

File: Kino/AppDelegate.swift [hunk 2]
```diff
@@ -7,30 +7,42 @@
...
+        tabBarController.viewControllers = [movies, ticket, fav]
+        
+        let navigationController = UINavigationController(rootViewController: tabBarController)
+
+        tabBarController.navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "list.star"), style: .plain, target: self, action: #selector(leftButtonTapped))
+        tabBarController.navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "bell.badge"), style: .plain, target: self, action: #selector(rightButtonTapped))
+        tabBarController.navigationItem.title = "KinkiNo"
+        
+        window = UIWindow(frame: UIScreen.main.bounds)
+        window?.rootViewController = navigationController
```
Comment (Darya-Astapava) [inline Kino/AppDelegate.swift]: There is no reason to create navigation controller with tab bar, you can directly assign tabBarController to the rootViewController

File: Kino/MainScreen/View/FavouritesViewController.swift [hunk 3]
```diff
@@ -0,0 +1,18 @@
+//
+//  FavouritesViewController.swift
+//  Kino
+//
+//  Created by Viktoriya Polozova on 15/08/2024.
+//
+
+import UIKit
+
+class FavouritesViewController: UIViewController {
```
Comment (Darya-Astapava) [inline Kino/MainScreen/View/FavouritesViewController.swift:10]: favorites

File: Kino/MainScreen/View/HeaderView.swift [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+//
+
+import UIKit
+
+class HeaderView: UIView {
+    
+    @IBOutlet weak var label: UILabel!
+    @IBOutlet weak var button: UIButton!
+    
+    // Метод для загрузки из XIB
```
Comment (Darya-Astapava) [inline Kino/MainScreen/View/HeaderView.swift:15]: all comments should be in English

File: Kino/MainScreen/View/FavouritesViewController.swift [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+            super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+        }
```
Comment (Darya-Astapava) [inline Kino/MainScreen/View/FavouritesViewController.swift:13]: alignment here and further

File: Kino.xcodeproj/xcuserdata/viktoriyapolozova.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 6]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Darya-Astapava) [inline Kino.xcodeproj/xcuserdata/viktoriyapolozova.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:1]: Add to gitignore

**Review by Darya-Astapava — CHANGES_REQUESTED** [review]

File: Kino/MainScreen/View/FavouritesViewController.swift (hunk 3 above)
Comment (niewyspana) [reply Kino/MainScreen/View/FavouritesViewController.swift:10]: favourites in British / favorites in American

File: Kino/MainScreen/View/HeaderView.swift (hunk 4 above)
Comment (niewyspana) [reply Kino/MainScreen/View/HeaderView.swift:15]: got it

File: Kino/MainScreen/View/FavouritesViewController.swift (hunk 5 above)
Comment (niewyspana) [reply Kino/MainScreen/View/FavouritesViewController.swift:13]: done

File: Kino/AppDelegate.swift (hunk 2 above)
Comment (niewyspana) [reply Kino/AppDelegate.swift]: done

**Pull request merged**